private/buf/bufcli/bufcli.go

```diff
@@ -249,6 +254,20 @@ Symlinks are never followed in Windows.`,
 	)
 }
 
+var allVisibiltyStrings = []string{
+	publicVisibility,
+	privateVisibility,
+}
```

Comment:
Nit: order globals towards top of the file, see https://www.notion.so/bufbuild/Go-Code-Standards-ff40382e6b104c1097a017f9565d54db#c6aeeed92adf40e189d7323ba0ccd11d